### ISSUE TITLE
Redesign UI for location popups

### ISSUE THREAD

**Issue by mycarrysun** [SCODEMeetup/community-services-locator-ui#59]

**Comment by OhioAdam** [comment]
What are some details for this?

**Comment by mycarrysun** [comment]
I had wanted to make them "prettier"....follow material design guidelines and use a white background with a card shadow and rounded corners. Also change the colors of the text to look more pleasing to the eye. Basically mimic the styles that Google maps uses when pulling up locations.

**Comment by mihiramin89** [comment]
yup all for changing this. The original design was to minimize the use of
colors and we were already using those colors in the application.
